pycytominer/tests/test_cyto_utils/test_load.py

```diff
@@ -196,3 +197,22 @@ def test_load_npz():
         load_npz_locations(
             example_npz_file_locations, location_x_col_index=0, location_y_col_index=2
         )
+
+def test_infer_plate_files():
+
+    # file paths
+    csv_file = "../test_data/cytominer_database_example_data/test_SQ00014613.csv.gz"
+    parquet_file = "../test_data/cytominer_database_example_data/test_SQ00014613.parquet"
+
+    # checking parquet file
+    check_pass = is_path_a_parquet_file(parquet_file)
+    check_fail = is_path_a_parquet_file(csv_file)
+
+    # checking if the correct booleans are returned
+    assert(check_pass, True)
+    assert(check_fail, False)
+
+    # loading in pandas dataframe from parquet file
```

Comment:
Is it worth also loading the csv file with `load_profiles()` and `pd.testing.assert_frame_equal(csv_df, parquet_df)`?